### ISSUE TITLE
Incorrect Error Messages 

### ISSUE THREAD

**Issue by Mochnant** [nedbrek/Atlantis#4]
Some errors are needlessly alarming and should be removed.

> Errors during turn:
> Warriors (295): MOVE: Unit has insufficient movement points;
> remaining moves queued.

The "insufficient move" message is not an error, but is rather part of normal gameplay (queuing moves).

> Errors during turn:
> Horse Trainers (262): GIVE: Nonexistant target (275).

The "nonexistent target" message is not generally helpful, and at least one tool has a flag to ignore them.  When making long term repeating orders, it is often easier to leave an @ GIVE message every turn than try and figure out which turn the wagons/etc will be there.  At very least it could be downgraded to a warning.

**Comment by Mochnant** [comment]
Additional spam error messages:

```
Errors during turn:
Armorers (146): Must be combat ready to be on guard.
Weaponsmiths (147): Must be combat ready to be on guard.
```

These are generated when a new unit is created by a unit that is on guard.  These should either be suppressed when new units are forming.

**Comment by nedbrek** [comment]
Just pushed a patch to move the move/sail error messages to events.

**Comment by Mochnant** [comment]
Also fixed with 7199297a96fc2a19949448d1505ce5e4aa9a8103.
